src/D2L.Security.OAuth2/Validation/Request/IRequestAuthenticator.cs

```diff
@@ -1,13 +1,8 @@
 ﻿using System.Net.Http;
 using System.Threading.Tasks;
 using D2L.Security.OAuth2.Principal;
-using D2L.Security.OAuth2.Validation.AccessTokens;
 
 namespace D2L.Security.OAuth2.Validation.Request {
-	/// <summary>
-	/// An abstraction for authenticating access tokens that works at the request level
-	/// rather than the token level (see <see cref="IAccessTokenValidator"/>)
-	/// </summary>

```

Comment:
IAccessTokenValidator is public, but the D2LPrincipal construction isn't. I didn't think this comment was completely accurate before and this change made it totally inaccurate.